pkg/client/clientset/versioned/fake/register.go

```diff
@@ -23,14 +23,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 // AddToScheme adds all types of this clientset into the given scheme. This allows composition
 // of clientsets, like in:
 //
-//   import (
```

Comment:
this is .. b/c of go versions: 

I guess ... we should discuss this? doing `code gen` w/ newer `golang` ?